bin/hipcc

```diff
@@ -215,7 +228,7 @@ if ($HIP_PLATFORM eq "clang") {
     }
 
     if ($HIP_RUNTIME eq "HCC" ) {
-      $HSA_PATH=$ENV{'HSA_PATH'} // "/opt/rocm/hsa";
+      $HSA_PATH=$ENV{'HSA_PATH'} // "$HIP_PATH/../hsa";
```

Comment:
Probably want ```$ROCM_ROOTDIR/hsa``` here.

Reply:
Why? What's wrong with $HIP_PATH/../hsa? You don't a separate variable for every directory.
The point of this fix is to not hardcode "/opt/rocm". This is not about inventing variable names for every directory or item used.

Reply:
In my understanding , as per current code, $HIP_PATH/../hsa need not be valid always. I can clone and build and install HIP - standalone- at any path today and set that as HIP_PATH.

Reply:
@gargrahul Unfortunately, "your understanding" is not addressed by this PR. Your scenario will get screwed whether $ROCM_ROOTDIR (which is derived from $HIP_PATH) is used or not.
Let's keep it simple.

Reply:
"My understanding" is captured here - https://github.com/ROCm-Developer-Tools/HIP/blob/master/INSTALL.md#building-hip-from-source- That's how HIP is being used for now.

I am NOT against keeping it simple though this needs to be thought are we going away where can we build and install HIP?

Reply:
The changes made are consistent with the web link as the default locations of hip and hsa are preserved. Please read the default paths and environment variables section under AMD-hcc.
Regarding,  "going away where can we build and install HIP" , I have no idea what you are talking about as that is orthogonal to this change. Once again, I ask we keep it simple.

Reply:
Rather than conjuring up what-if scenarious, please try the code changes and report problems that the original code addressed better.